oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/DelegatingOAuth2TokenValidator.java

```diff
@@ -16,17 +16,16 @@
 
 package org.springframework.security.oauth2.core;
 
+import org.springframework.util.Assert;
```

Comment:
Since these are just whitespace changes, let's leave this out of the PR.

Reply:
Fixed